core/domain/topic_services.py

```diff
@@ -312,6 +312,91 @@ def update_topic(
     create_topic_summary(topic.id)
 
 
+def delete_skill(user_id, topic_id, skill_id):
+    """Removes skill with given id from the topic.
+
+    Args:
+        user_id: str. The id of the user who is performing the action.
+        topic_id: str. The id of the topic from which to remove the skill.
+        skill_id: str. The skill to remove from the topic.
+    """
+    topic = get_topic_by_id(topic_id)
+    new_skill_ids = topic.delete_skill(skill_id)
+    change_list = [topic_domain.TopicChange({
+        'cmd': 'update_topic_property',
+        'property_name': 'skill_ids',
+        'old_value': topic.skill_ids,
```

Comment:
Should do copy.deepcopy() to avoid binding. Ditto below.

(Unless the TopicChange constructor already does that?)

Reply:
Done.